ui/admin/app/components/form/host-set/azure/index.hbs

```diff
@@ -168,7 +168,7 @@
     @value={{@model.filter}}
     @label={{t 'resources.host-set.form.filter.label'}}
     @helperText={{t 'resources.host-set.form.filter.help'}}
-    @linkText={{t 'form. tiltes.learn-more'}}
+    @linkText={{t 'titles.learn-more'}}
```

Comment:
I do not have much context on translations, perhaps @susmitha can help with this.

But looking for the same pattern in our codebase, in the past you add this same translation within `actions` which makes more sense to me.

Why is this one in the root of translations? And why we can't re-use the previous pattern you set?

Reply:
ah, my bad! I noticed a typo here and fixed it. it should be actions. 
but yes, more context on translations would be great.